plugin/src/test/java/org/opensearch/ml/rest/RestMLGuardrailsIT.java

```diff
@@ -423,7 +427,11 @@ protected Response registerRemoteModelWithLocalRegexGuardrails(String name, Stri
             + "      ],\n"
             + "      \"regex\": [\"regex1\", \"regex2\"]\n"
             + "    }\n"
-            + "  }\n"
+            + "},\n"
```

Comment:
the IT only covers the general model general interface, please add the IT for aws llm interface later

Reply:
This is to resolve IT failure, we have complete interface IT: https://github.com/opensearch-project/ml-commons/pull/2394